src/main/java/page_objects/RegisterPage.java

```diff
@@ -14,6 +14,9 @@
     private By txtPid = By.id("pid");
     private By btnRegister = By.cssSelector("input[type='submit']");
     private By lblSuccessMessage = By.cssSelector("#content p");
+    private By lblErrorMessage = By.cssSelector("#content>p[class = 'message error']");
```

Comment:
Please specify what error is, like the 2 below errors (ex: lblErrorFormMessage)

Reply:
I fixed it, help me check it. Thanks so much